### PR TITLE
TRIUMPH - Secquipment storage fix - slight HoS office edit - sec shuttle replacement

### PR DESCRIPTION
**Pull request by Nut2**
## About The Pull Request
Fixes the security equipment storage once again with hopefully no conflicts this time. It adds plate carriers, helmet cameras, survival knives, ETC, while also removing some uneeded items. 

Removed some unneeded items from the HoS's office

replaced shuttles in the podbay with barons 

## Why It's Good For The Game
good riddance

### CODE REVIEW

File: maps/triumph/levels/deck4.dmm [hunk 1]
```diff
@@ -6334,9 +6334,13 @@
...
 "eGK" = (
 /obj/structure/table/woodentable,
-/obj/item/radio/off,
-/obj/item/megaphone,
-/obj/item/storage/box/glasses/rocks,
+/obj/item/radio/off{
+	pixel_x = -7;
+	pixel_y = 2
+	},
+/obj/item/megaphone{
```
Comment (TheLordME) [inline maps/triumph/levels/deck4.dmm:6341]: you should already get a megaphone in your locker

File: maps/triumph/levels/deck4.dmm [hunk 2]
```diff
@@ -12899,6 +12942,10 @@
 /obj/structure/cable/green{
 	icon_state = "1-8"
 	},
+/obj/item/clothing/under/oricon/utility/fleet/security,
```
Comment (TheLordME) [inline maps/triumph/levels/deck4.dmm:12945]: milrp

Comment (Nut2) [reply maps/triumph/levels/deck4.dmm:12945]: spawn-with-able item put into locker (sili said sure)

File: maps/triumph/levels/deck4.dmm (hunk 1 above)
Comment (Nut2) [reply maps/triumph/levels/deck4.dmm:6341]: moved a spare one that was already there

**Review by TheLordME — APPROVED** [review]

**Pull request merged**